UselessMine.cpp

```diff
@@ -323,6 +344,28 @@ void UselessMine::Event (bz_EventData *eventData)
                 else if (mine.defused)
                 {
                     dieData->killerID = mine.defuserID;
+
+                    if (playerID != mine.owner)
+                    {
+                        if (!defusalMessages.empty())
+                        {
+                            // The random number used to fetch a random taunting defusal message
+                            int randomNumber = rand() % defusalMessages.size();
+
+                            // Get a random defusal message
+                            std::string defusalMessage = defusalMessages.at(randomNumber);
+                            bz_sendTextMessage(BZ_SERVER, BZ_ALLUSERS, formatMineMessage(defusalMessage, bz_getPlayerCallsign(mine.defuserID), owner).c_str());
+                        }
+                        else
+                        {
+                            // Let the BD player know that they killed the owner
+                            bz_sendTextMessagef(BZ_SERVER, dieData->killerID, "You defused %s's mine", owner);
+
+                            // Let the owner know that they killed the BD player
+                            bz_sendTextMessagef(BZ_SERVER, playerID, "You were killed by %s's mine defusal", victim);
```

Comment:
Instead of `playerID` make it `mine.owner` and I think `victim` needs to be replaced with `bz_getPlayerCallsign(mine.defuserID)` because `victim` here is the callsign of the player who just died